final-project/README.md

```diff
@@ -5,6 +5,45 @@ This application allows users to sign up for an account, sign in, record and man
 
 The server and client are both written on Go, and run on Docker.
 
+## Project Directory
+```
+final-project/
+│   Taskfile.yml
+│   go.mod
+│   main.go                     - main file for running the server
+│   README.md
+│   routes.go
+│
+└───client/                     - module for client app
+│   
+└───deploy/
+│   │
+│   └───dev/
+│       │   
+│       └───data/
+│       │   │        
+│       │   └───data.json       - sample data for testing dev
+│       │ 
+│       │   .env                - environment variables for dev
+│       └───docker-compose.yml
+│   
+└───server/
+    │   auth/                   - package for user registration and authentication
+    │   categories/             - package for handling categories of transactions
+    │
+    └───data/
+    │   │   
+    │   └───filebased/          - package for file-based DB implementation
+    │   │   
+    │   └───redis/              - package for Redis store implementation  
+    │   │
+    │   └───data.go             - file for data layer interface implementation 
+    │
+    │   sessions/               - package for handling user sessions
+    │   transactions/           - package for handling user transactions
+    └───Dockerfile
```

Comment:
I suggest to move that Dockerfile in "/deploy".
At the moment we don't know it will be the same file dev/stage/prod.

Reply:
My client app also has its own Dockerfile. Do I also move it to `deploy/`?
If yes, can the structure be like this:
In `deploy/dev/`, I have `client/` and `server/`. Dockerfile of client goes to `client/`, while Dockerfile of server and `data/` goes to `server/`

Reply:
Yep you can do that, of course.